ipywidgets/widgets/widget.py

```diff
@@ -223,6 +272,10 @@ def open(self):
             if buffers:
                 # FIXME: workaround ipykernel missing binary message support in open-on-init
                 # send state with binary elements as second message
+                # TODO: if this gets fixed, _split_state_buffers does not need to have a seperate
```

Comment:
Looks like it was added about 1.5 years ago: https://github.com/ipython/ipykernel/commit/64c69213be7629bbf8735a7fabdb4e47977775c9